apps/studio/components/interfaces/Auth/Hooks/AddHookDropdown.tsx

```diff
@@ -85,14 +88,14 @@ export const AddHookDropdown = ({
         {nonEnterpriseHookOptions.length > 0 && <DropdownMenuSeparator />}
 
         <div className="bg-surface-200 p-1 -mt-2">
-          {subscription?.plan.id !== 'enterprise' && (
+          {!isTeamsOrEnterprisePlan && (
             <DropdownMenuLabel className="grid gap-1 bg-surface-200">
-              <p className="text-foreground-light">Enterprise plan required</p>
+              <p className="text-foreground-light">Teams or Enterprise plan required</p>
               <p className="text-foreground-lighter text-xs">
                 The following hooks are not available on{' '}
                 <a
                   target="_href"
-                  href="https://forms.supabase.com/enterprise"
+                  href="https://supabase.com/dashboard/org/_/billing"
```

Comment:
Given we are in an org context - this can use the org slug directly instead of the generic `_` org redirect